src/main/webapp/app/entities/event-activity/event-activity.tsx

```diff
@@ -99,7 +99,9 @@ export class EventActivity extends React.Component<IEventActivityProps, IEventAc
                   <span className="card-item d-block mb-2">
                     <span>
                       <Translate contentKey="clubmanagementApp.eventActivity.duration">Duration (In Day)</Translate>:{' '}
```

Comment:
should be using durationInDay instead of duration

Reply:
I think it should be duration since the duration is not longer based on "day" as before. The mistake here is the text in the `<Translate>` tag should be "Duration".

Reply:
But thanks for bringing it up

Reply:
Resolved in 2d6dc9ee70121543ae0be8433fcfc8f9801146cc